src/profile/v100/address/non_fungible_local_id.rs

```diff
@@ -181,6 +152,14 @@ mod tests {
         );
     }
 
+    #[test]
+    fn from_invalid_str_error() {
+        assert_eq!(
+            "#value#".parse::<SUT>(),
```

Comment:
prefer using something which is more obviously invalid, e.g. `"super invalid string!!!!!"`